src/pages/Subscription/Subscription.scss

```diff
@@ -0,0 +1,49 @@
+.info {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
+
+  h1 {
+    font-size: 30px;
+    margin-bottom: 10px;
+
+    &::after {
+      content: ".";
+      color: blue;
+    }
+  }
+
+  p {
+    font-size: 40px;
+    margin: 10px 0px !important;

```

Comment:
Do you need this `!important`?

Reply:
yes, i tryed without it and doesn't work.